src/AppLayout.vue

```diff
@@ -43,11 +44,13 @@ import { useUser } from "./state/user.js";
 // Components
 import AppHeader from "./components/specific/app/app-header/AppHeader.vue";
 import AppNotification from "./components/specific/app/app-notification/AppNotification.vue";
+import AppSidePanel from "./components/specific/app/app-side-panel/AppSidePanel.vue";
 
 export default {
   components: {
     AppHeader,
-    AppNotification
+    AppNotification,
+    AppSidePanel
```

Comment:
it seems that a comma is missing, is eslint well configured ?

Reply:
Yes, trailing commas are disabled on this repo.